app/controllers/components/export_csv.php

```diff
@@ -127,7 +127,7 @@ function createCsv($params, $event)
      * @access public
      * @return void
      */
-    function generateHeader($params, $event, $peerEval = 1)
+    function generateHeader($params, &$event, $peerEval = 1)
```

Comment:
Removed the instances of pass by reference. Somehow I didn't know PHP implemented copy-on-write